src/CMakeLists.txt

```diff
@@ -400,7 +400,7 @@ if(USE_HEIF)
   if(NOT TARGET heif)
     find_package(libheif MODULE)
```

Comment:
add 1.13.0 for the "module" search, it'll give a more meaningful message in the log

(doesn't work for the "config" search above as it looks for exact version match, and not for greater or equal...)

Reply:
@g2p : Just a ping in case you missed this comment from @kmilos. I'll then merge for 4.2. TIA.

Reply:
Like this?  I don't know cmake much.

Reply:
Yep, good to go!